delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/config/ConfigReaderInstances.scala

```diff
@@ -36,6 +38,28 @@ trait ConfigReaderInstances {
         .map(uri => BaseUri(uri))
     )
 
+  implicit final val serviceAccountConfigReader: ConfigReader[ServiceAccountConfig] =
+    ConfigReader.fromCursor { cursor =>
```

Comment:
Do we need such a verbose way to define the `ConfigReader` for `ServiceAccountConfig `? The derived one does not work?

Reply:
ServiceAccountConfig holds a Subject so we have to make one out the two fields.

Reply:
Can't you just create a `ConfigReader[Subject]`  for this? And then the `ConfigReader[ServiceAccountConfig]` can be derived

Reply:
The conf file would have to look a little different and not so straightforward.

And if I create a ConfigReader for the Subject, it will lead more or less to the same verbosity